### ISSUE TITLE
Need new icons for landmark applications

### ISSUE THREAD

**Issue by pwpatton** [infor-design/design-system#391]
In landmark we have a set of lpl icon names that we try and map to the current icon set. There are few we are missing so requesting those here.

![image](https://user-images.githubusercontent.com/4737815/61890182-8f2a8280-aecc-11e9-9133-0acb976044d9.png)

https://jira.lawson.com/secure/attachment/1117285/LPL%20Icon%20to%20SoHo%20Icon.pdf

https://jira.lawson.com/browse/LMCLIENT-21862

**Comment by tmcconechy** [comment]
We have some similar icons http://latest-enterprise.demo.design.infor.com/components/icons/example-index.html

- apply : icon-invoicing looks identical
- award: not sure if anything suitable
- calculate: icon-calculator  looks identical
- claim: not sure if anything suitable
- finalize: not sure if anything suitable
- finalize: not sure if anything suitable (#icon-new-payment-request doesnt seem right)
- transfer: not sure  maybe icon-send?
- validate: we have these as #icon-success

**Comment by pwpatton** [comment]
Here are the lpl icon names landmark support for our developers and how they map to a soho/uplift icon.

Here are the landmark LPL icon names. These can be used on toolbar buttons
```
ActionIcon ::=
    ( add
	| apply
	| approve
	| award
	| calculate
	| chart
	| claim
	| copy
	| create
	| delete
	| edit
	| export
	| filled star
	| filter
	| finalize
	| open
	| outlined star
	| pdf
	| preview
	| print
	| process
	| reject
	| release
	| request
	| save
	| search
	| send
	| submit
	| transfer
	| undo
	| validate
	)
```

Here is what we map each of these to right now.
```
  /**
   * Returns a soho/uplift icon given an LPL action icon.
   * @param icon the icon to map to a soho/uplift icon
   */
  protected lplIconToSohoXI(icon: LMIcon): string | null {
    switch (icon) {
      case 'add':             return 'add';
      case 'apply':           return 'invoicing';             // todo NEED ICON - icons not consistent between soho and uplift
      case 'approve':         return 'check';
      case 'award':           return 'new-solid';             // todo NEED ICON - icon doesnt' look right with white star - maybe make transparent star
      case 'create':          return 'new-document'; // add
      case 'calculate':       return 'calculator';             // todo NEED ICON - uplift calculate doesn't look like a calculator
      case 'chart':           return 'line-chart';
      case 'claim':           return 'flag';             // todo NEED ICON - icon is red, needs to be like other icons - fixed
      case 'copy':            return 'copy';
      case 'delete':          return 'delete';
      case 'edit':            return 'edit';
      case 'export':          return 'export-2';
      case 'filled star':     return 'star-filled';
      case 'filter':          return 'filter';
      case 'finalize':        return '';                 // todo NEED ICON
      case 'open':            return 'open-folder';
      case 'outlined star':   return 'star-outlined';
      case 'pdf':             return 'pdf-file';
      case 'preview':         return 'update-preview';
      case 'print':           return 'print';
      case 'process':         return 'process';
      case 'refresh':         return 'refresh';
      case 'release':         return 'invoice-released';
      case 'request':         return '';                  // todo NEED ICON
      case 'reject':          return 'reject';
      case 'save':            return 'save';
      case 'search':          return 'search';
      case 'send':            return 'send';
      case 'sort-ascending':  return 'sort-down';
      case 'sort-descending': return 'sort-up';
      case 'submit':          return 'send-submit';
      case 'helperList':      return 'amend';
      case 'transfer':        return 'send';             // todo NEED ICON
      case 'undo':            return 'undo';
      case 'validate':        return 'success';          // todo NEED ICON
      default:
        return null;
    }
  }
```

**Comment by pwpatton** [comment]
Another issue is since save-new doesn't seem to have an uplift icon. I know this was deprecated, but we really rely on it on our forms and cannot deprecate that functionality.

![image](https://user-images.githubusercontent.com/4737815/64350017-9198f580-cfbd-11e9-9265-4cc4bc28a741.png)

**Comment by jamie-norman** [comment]
Some icons were "retired" without understanding the regression impact, perhaps. Or at least there isn't a readily available replacement without app developers having to go in and re-do a lot of LPL where something like save-new exists

**Comment by kevinwhitedesign** [comment]
Got it

**Comment by kevinwhitedesign** [comment]
Apply Icon
We have a doc with check icon that for some reason did not get into the icon system. Adding it in for you guys.

Award
Creating an outlined version.

Finalize
We suggest mapping to the ‘Check’ icon.

Request
We suggest mapping to the ‘Comment’ icon.

Calculator
Working on a redesign of this icon.

Save New
Working on an uplift style of this icon. I think this warrants more information for IDS to support it however. Our team discussed a bit internally and are wondering about icon usage VS. text and also the ux of saving in LMK. Shall we investigate this?

Check out these mocks below. 

![Screenshot 2019-09-06 16 23 45](https://user-images.githubusercontent.com/6478704/64458200-b9be4c80-d0c2-11e9-940f-94460448a410.png)
![Screenshot 2019-09-06 16 24 30](https://user-images.githubusercontent.com/6478704/64458241-cf337680-d0c2-11e9-8665-f75c4bd792d1.png)

**Comment by kevinwhitedesign** [comment]
Let me know if we can move forward

**Comment by pwpatton** [comment]
@kevinwhitedesign We are already using the 'check' icon for Approve. Should we use a different icon for that if we use the 'check' icon for Finalize?

**Comment by kevinwhitedesign** [comment]
Not If the functionality is similar or if the action is a similar experience: User says: "this is okay, I'm clicking this 'done' or 'submit' button". 

If this is so, then the icon works for both. Also, if the text is submit or done the icon would also work. It's pretty universally recognizable.

**Comment by pwpatton** [comment]
@kevinwhitedesign @jamie-norman I guess we'd need to ask the apps groups if they use these 2 as interchangeable.

**Comment by pwpatton** [comment]
![image](https://user-images.githubusercontent.com/4737815/64539934-fcf80580-d2e4-11e9-926b-8db70f1e5908.png)

**Comment by pwpatton** [comment]
@kevinwhitedesign I think we should move forward with the icons you have listed above. My only concern is using the 'check' icon for both finalize and approve.

**Comment by lipetzan** [comment]
Agree Finalize and Approve should differ.  

Could Finalize be a checklist looking thing with all things checked?  Like this, but forget the Approved part.
![image](https://user-images.githubusercontent.com/42155063/64553917-ead79080-d2ff-11e9-9cb5-bede6d6cf7dd.png)

**Comment by kevinwhitedesign** [comment]
Alright the jury's in. For sure, I'll come up with some shapes.

**Comment by kevinwhitedesign** [comment]
Ok guys, here's the set again. Just say the word and poof, icons you will have.
![Screenshot 2019-09-10 10 53 20](https://user-images.githubusercontent.com/6478704/64624856-44e85c80-d3b9-11e9-8080-869ba2af572f.png)

**Comment by kevinwhitedesign** [comment]
Just for reference, the actual IDS icon names will be the bottom row on this screenshot. So you can map more easily. Submitting to IDS today.

<img width="687" alt="Screenshot 2019-09-11 12 53 07" src="https://user-images.githubusercontent.com/6478704/64717883-1ccd2c00-d493-11e9-8892-4f49c8dd027e.png">

**Comment by kevinwhitedesign** [comment]
I just submitted a pull request, waiting on IDS.

**Comment by EdwardCoyle** [comment]
I just merged this into `master`.

@tmcconechy we will probably need to make an `ids-identity` release for this.

**Comment by kevinwhitedesign** [comment]
Updated this yesterday, fixed some naming and added backwards compatibility.

**Comment by tmcconechy** [comment]
I also pushed the needed code changes https://github.com/infor-design/enterprise/pull/2787

**Comment by janahintal** [comment]
Hi @tmcconechy , http://master-enterprise.demo.design.infor.com/components/icons/example-index.html?theme=uplift&variant=dark&colors=0053B1 still dont have Save new icon.

**Comment by janahintal** [comment]
QA Passed. Will move to Done. Thanks 
Verified in http://master-enterprise.demo.design.infor.com/components/icons/example-index.html?theme=uplift&variant=contrast&colors=003066 & http://master-enterprise.demo.design.infor.com/components/icons/example-index.html Across all platforms

**Comment by tmcconechy** [comment]
@janahintal I see the icon here as highlighted as mentioned just FYI

![Screen Shot 2019-09-30 at 10 05 20 AM](https://user-images.githubusercontent.com/814283/65886169-d9ffc500-e369-11e9-814b-1633d07231d0.png)
